scenario/elasticopenhub.go

```diff
@@ -232,6 +245,19 @@ func (openHub ElasticOpenHubSettings) Execute(sessionState *session.State, actio
 	}
 }
 
+func sendEvaluation(sessionState *session.State, actionState *action.State, host string, resources []elasticstructs.EvaluationResource) {
```

Comment:
Not a huge fan of this name since it has nothing to do with app evaluation :P I know the endpoint is called evaluation. Maybe sendPolicyEvaluation?

Reply:
app evaluation? what's that? :p

Reply:
🐳